shared/javascripts/options.js

```diff
@@ -63,6 +63,16 @@ if (Privly === undefined) {
   }
   Privly.options = {};
 
+  // CommonJS Module
+  if (typeof module !== "undefined" && module.exports) {
+    module.exports.options = Privly.options;
+    // load dependencies
+    ls = require("./local_storage.js").ls;
+    Privly.storage = require("./storage.js").storage;
```

Comment:
OK, `n!`. Still, this seems odd from a performance perspective. Perhaps it would be good to check whether the object is already defined, then `require` it when needed. These variables are in global namespace so they can be shared.